### PR TITLE
CGE-64_Serve_Static_Resources_with_Spring

### PR DESCRIPTION
**Pull request by 21November**
Create endpoints to display pages with this behavior:

URL | Page Name
/registration | registration.html
/search | result.html
/profile | profile.html
/profile/settings | settings.html
/profile/favorites | favorites.html
/product/info | info.html
/product/create | create.html

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -21,7 +21,8 @@ spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
 spring.jpa.hibernate.ddl-auto=update
 
 ## Spring View Resolver
-spring.mvc.view.prefix=html/
+spring.resources.static-locations=classpath:/static/html/, classpath:/static
+##spring.mvc.view.prefix=
```
Comment (rpkyrych) [inline src/main/resources/application.properties]: We don't need this line anymore - you should remove it.

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -21,7 +21,8 @@ spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
 spring.jpa.hibernate.ddl-auto=update
 
 ## Spring View Resolver
-spring.mvc.view.prefix=html/
+spring.resources.static-locations=classpath:/static/html/, classpath:/static
```
Comment (rpkyrych) [inline src/main/resources/application.properties]: :+1:

File: src/test/java/com/hillel/items_exchange/controller/PageControllerTest.java [hunk 3]
```diff
@@ -25,4 +25,20 @@ public void uponRequestTemp_redirectsToThePageTemp() throws Exception {
                 .andExpect(status().isOk())
                 .andExpect(view().name("temp"));
```
Comment (rpkyrych) [inline src/test/java/com/hillel/items_exchange/controller/PageControllerTest.java]: You should completely remove this test as well.

File: src/main/java/com/hillel/items_exchange/controller/PageController.java [hunk 4]
```diff
@@ -9,4 +9,15 @@
     public String getTemp() {
         return "temp";
```
[the remark this answers is not in the document]
Comment (21November) [reply src/main/java/com/hillel/items_exchange/controller/PageController.java]: remote method getTemp.

File: src/test/java/com/hillel/items_exchange/controller/PageControllerTest.java (hunk 3 above)
Comment (21November) [reply src/test/java/com/hillel/items_exchange/controller/PageControllerTest.java]: test remotely

**Pull request merged**